src/index.js

```diff
@@ -4,10 +4,11 @@ import { createConfig } from './create-config';
 import module from './module';
 import { setEndpoint, removeQueryParams } from './helpers';
 
-
+// TODO: avoid to export undefined globals, redefined somewhere else
```

Comment:
Do you have a concrete idea to avoid this pattern? I've looked at similar plugins and haven't seen any better idea